### ISSUE TITLE
Advanced Search - 2.4.6 Headings and Labels - 1 skipped heading level is present on the Search field title

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9867]
As a user reliant on a screen reader, I expect heading levels to be in a sequential order so that I can form a mental image of the structure of the page and be able to skip to content quickly.

**Linked work item:**
#9386 

**Expected Result:**
Heading levels in the page should be sequential so that screen reader users are able, not only to form a mental image of the structure of the page, but also to skip to content they want without worrying that they have missed something important.

**Actual Result:**
The heading level here is at h4, meaning that h3 has been skipped and might cause users to believe they have missed important information on the page.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/0456ff29-ede8-4ed9-a4b3-bfc5a6129276)

**Comment by apeters** [comment]
I think we should just remove all heading levels as it's too hard manage consistent levels

**Comment by imogenroberts** [comment]
This might actually work - as long as headings are clear and descriptive of contents they don't have to be numbered (not sure how this works from a dev standpoint). See below for full interpretation from WCAG:

![image](https://github.com/archesproject/arches/assets/77161085/d0c1ceed-ff0c-40f4-8dfd-161487e49e90)

**Comment by johnatawnclementawn** [comment]
@apeters @imogenroberts This should be resolved when #9829 is merged in.

Alexei - I see there are merge conflicts there. Those are from primarily caused by the em/rem font sizes & focus shifting PR's being merged into accessibility... so I'll just need to spend a bit of time making sure those previous changes are properly migrated into #9829

**Comment by apeters** [comment]
@imogenroberts this should be fixed in the latest, can you retest to confirm?

**Comment by imogenroberts** [comment]
RETEST: Skipped heading level on 'Search Facets' heading has now been fixed.

**Evidence:**
![Advanced Search - 2 4 6  Headings and Labels FIXED](https://github.com/archesproject/arches/assets/77161085/9590073b-80b4-4138-a067-1ecf323b9f0e)
